decorators/ng-module/metadata-handlers/import.ts

```diff
@@ -0,0 +1,22 @@
+export default function importHandler(ng1Module: any, ng1ModuleIds: any, imports: any) {
+    const modules = imports.filter(({ $inject }: any) =>
+        !($inject && $inject.find((i: any) => i === "$stateProvider")));
+
+    const ng1RouterConfig = imports.find(({ $inject }: any) =>
+        $inject && $inject.find((i: any) => i === "$stateProvider"));
```

Comment:
Very tricky and unreliable. We should think of a better way to differentiate router config